source/newpmcmember.md

```diff
@@ -1,47 +1,76 @@
 ---
-title: New PMC member
+title: Inviting a New PMC member
 tags: ["pmc", "pmcmembers", "election"]
 ---
 
-## New PMC member process
+This document describes a typical Apache project's process for handling the
+vote to add a new PMC (Project Management Committee) member - when it is
+separate from becoming committer.
 
-This section describes a typical Apache project's process for handling the
-vote to add a new PMC member - when it is separate from becoming committer.
-Templates mentioned in the process appear later in this document. If your PMC
-adds automatically committers as PMC members, the process for doing both at the
-same time is described in the [New committer](https://community.apache.org/newcommitter.html)
-The [Contributor Ladder](https://community.apache.org/contributor-ladder.html) helps explain different roles.
+Templates mentioned in the process appear [#NewPMCMember-EmailTemplates](later in 
+this document). If your PMC adds automatically committers as PMC members, the process
```

Comment:
```suggestion
this document). If your PMC automatically adds committers as PMC members, the process
```

Minor nit but I believe `add` is being used transitively here and the adverb comes first.